dags/copy_deduplicate.py

```diff
@@ -146,7 +148,11 @@
         task_id="copy_deduplicate_main_ping",
         target_project_id="moz-fx-data-shared-prod",
         billing_projects=("moz-fx-data-shared-prod",),
-        only_tables=["telemetry_live.main_v4"],
+        only_tables=[
```

Comment:
The grouping makes sense... Could `copy_deduplicate_main_ping` be timing out now? Is it worth a separate DAG for these two tasks?

Reply:
It shouldn't timeout, these queries are running in chunks (`slices=20`).